components/async-container/async-container.js

```diff
@@ -0,0 +1,43 @@
+import '../loading-spinner/loading-spinner.js';
+import { AsyncContainerMixin, asyncStates } from './async-container-mixin.js';
+import { html, LitElement } from 'lit-element/lit-element.js';
+
+class AsyncContainer extends AsyncContainerMixin(LitElement) {
```

Comment:
I'm not sure whether we should create this component or not.  Perhaps it should just be a demo component...?

Reply:
I think I'd lean towards a demo component for now. The render logic around needing to specify things in all the slots and the fact that a loading spinner is being used seems somewhat specific to this particular use case. I could see other "async containers" wanting different things maybe? I dunno. Especially if the delay moves into the mixin, then this isn't really providing much extra value.